### PR TITLE
Admin merchants

### PR DESCRIPTION
**Pull request by garrettgregor**
merge admin merchants into main

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ gem "puma", "~> 5.0"
 gem "importmap-rails"
 
 # Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
-gem "turbo-rails"
+# gem "turbo-rails"
```
Comment (JCBeldo) [inline Gemfile:22]: Slow down there tiger

File: app/controllers/admin/merchants_controller.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+class Admin::MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.top_5_merchants
```
Comment (jcjurado3) [inline app/controllers/admin/merchants_controller.rb:3]: Nice use of the instance variables here and eliminating the use of calling the method on the view

File: app/views/admin/merchants/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<%= render partial: "admin/admin_nav" %>
+<br>
+<br>
+<br>
+<br>
+<br>
+<%= form_with model: @merchant, url: admin_merchant_path(@merchant) do |f| %>
```
Comment (jcjurado3) [inline app/views/admin/merchants/edit.html.erb:7]: great use of the form model here

File: config/routes.rb [hunk 4]
```diff
@@ -4,19 +4,18 @@
...
     resources :items, only: [:index, :show, :edit, :update, :new, :create], controller: "merchants/items"
     resources :invoices, only: [:index, :show], controller: "merchants/invoices"
     resources :invoice_items, only: [:update], controller: "merchants/invoice_items"
   end
-  
+
   resources :admin, only: [:index]
   namespace :admin do
-    resources :merchants, only: [:index]
+    resources :merchants, only: [:index, :show, :edit, :update, :new, :create]
```
Comment (jcjurado3) [inline config/routes.rb:18]: clean resources!

**Review by boomclear — APPROVED** [review]
looks good. Removed some comments for you.

**Review by JCBeldo — APPROVED** [review]
Let's Elfin' Go! One more Story and we can get this done!

File: spec/features/admin/merchants/update_spec.rb [hunk 5]
```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "admin merchant update", type: :feature do
+  load_test_data
```
Comment (jcjurado3) [inline spec/features/admin/merchants/update_spec.rb:4]: nice use of the load_test_data to call the test data

File: spec/rails_helper.rb [hunk 6]
```diff
@@ -36,6 +36,7 @@
   abort e.to_s.strip
 end
 RSpec.configure do |config|
+  require "helper_methods"
```
Comment (jcjurado3) [inline spec/rails_helper.rb:39]: noted!

**Review by jcjurado3 — APPROVED** [review]

File: app/controllers/admin/merchants_controller.rb (hunk 2 above)
Comment (garrettgregor) [reply app/controllers/admin/merchants_controller.rb:3]: 🙇🏻‍♂️

**Pull request merged**